### DISCUSSION TITLE
Welcome to autoDocstring Discussions!

### DISCUSSION THREAD

**Discussion by NilsJPWerner** [NilsJPWerner/autoDocstring#219] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ashimgiyanani** [comment]
Hi,

I am wondering if there exists a way to have multiple generate docString at the same time, for example I use an extensive docstring for classes while 1-2 liners for methods within the class. I would like to have a template for the same, similar ot one-line sphinx

Regards
Ashim
